README.md

```diff
@@ -63,6 +63,10 @@ You then need to refresh the JupyterLab page when your javascript changes.
 
 ## Developer notes
 
+To install developer tools:
+
+    $ pip install pytest black flake8 flake8-docstrings flake8-bugbear
```

Comment:
In the future it may be nice to have `pip install -r dev_requirements.txt` or something like that. Don't worry about it now.